### ISSUE TITLE
Counterparty API communication error

### ISSUE THREAD

**Issue by CryptoEconomize** [jdogresorg/freewallet-desktop#196]
Not sure how to resolve..  

![cpissue](https://github.com/jdogresorg/freewallet-desktop/assets/6612446/1c9e31c2-25f6-470d-8d57-98ae83e5684f)

**Comment by jdogresorg** [comment]
I would suggest joining the FreeWallet Telegram channel or twitter, as both places have already addressed this issue.

https://t.me/FreeWallet
https://x.com/FreeWallet

---
https://twitter.com/FreeWallet/status/1778229788605620371

Any FreeWallet users who are experiencing "API Communication" issues, it is because api.counterparty.io no longer supports HTTP requests.

If your getting the error, make sure your "Server Settings" match the  screenshot below

TLDR... Set "CP SSL" to "Yes"

![image](https://github.com/jdogresorg/freewallet-desktop/assets/376028/0629b566-5e0d-4bf9-8a29-9921dc5989e7)
![image](https://github.com/jdogresorg/freewallet-desktop/assets/376028/efc69b5b-26b0-4a16-ac53-654d51b74354)
